src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/UserTests.kt

```diff
@@ -0,0 +1,56 @@
+package org.vaccineimpact.api.blackboxTests
+
+import com.beust.klaxon.json
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.vaccineimpact.api.blackboxTests.helpers.RequestHelper
+import org.vaccineimpact.api.blackboxTests.helpers.TestUserHelper
+import org.vaccineimpact.api.blackboxTests.helpers.validate
+import org.vaccineimpact.api.blackboxTests.schemas.JSONSchema
+import org.vaccineimpact.api.db.JooqContext
+import org.vaccineimpact.api.db.direct.addUserWithRoles
+import org.vaccineimpact.api.models.permissions.ReifiedRole
+import org.vaccineimpact.api.models.Scope
+import org.vaccineimpact.api.models.permissions.PermissionSet
+import org.vaccineimpact.api.test_helpers.DatabaseTest
+
+class UserTests : DatabaseTest()
+{
+    @Test
+    fun `can get user by username`()
+    {
+        validate("/users/testuser") against "User" given {
+            it.addUserWithRoles("testuser", ReifiedRole("member", Scope.Specific("modelling-group", "group")))
```

Comment:
Is there a reason why `Scope.databaseScopeId` is nullable while `Scope.databaseScopeId` is not? Just thinking the inconsistency in the JSON for a global role ("scope_id" comes back as an empty string, "scope_prefix" as null) might be a little confusing to consumers of the API?